spine-godot/spine_godot/spine-cpp

```diff
@@ -0,0 +1 @@
+../../spine-cpp/spine-cpp
```

Comment:
I'm reviewing on GitHub, but this looks like a symlink? Can't have that I'm afraid.

Reply:
Will remove (sorry artifact from how we setup our projects)

Reply:
Oh, looking back I guess we did this so we could setup compilation of the module without an additional 'copy to build' step.   As the symlink is setup to spine-cpp location in the github repo.  Will still remove out of this PR though.